shards/gfx/render_step_impl.cpp

```diff
@@ -427,17 +459,17 @@ void setupRenderGraphNode(RenderGraphNode &node, NodeBuildData &buildData, const
 
   // Derive definitions from parameters
   for (auto &param : step.parameters.basic) {
-    baseFeature->shaderParams.emplace_back(param.first, param.second);
+    baseFeature->shaderParams.emplace(param.first, param.second);
   }
   for (auto &param : step.parameters.textures) {
     auto &textureFormat = param.second.texture->getFormat();
-    auto &entry = baseFeature->textureParams.emplace_back(param.first, textureFormat.dimension);
+    auto &entry = baseFeature->textureParams.emplace(param.first, textureFormat.dimension).first->second;
```

Comment:
emplace probably wrong, fails on re-insert

Reply:
Changed these to insert_or_assign, just in case